models/note.go

```diff
@@ -10,33 +10,34 @@ type NoteId int64
 type NoteType int
 
 const (
-	MARGINALIA NoteType = iota
+	UNCATEGORIZED NoteType = iota
```

Comment:
I felt like uncategorized label, should either be at the begging or at the end of the enum, since in some ways it stands apart from the other categories. Putting it at the end would cause its "number" to change as we added more categories.